wordpress-mobile/WordPress-iOS
# Remove Google domains promotional card from the dashboard

## Description
This PR disables the Google domains card in the dashboard.

## Testing Instructions

1. Install the Jetpack app
2. Scroll through the dashboard
3. Make sure the Google Domains card is not visible

## Regression Notes
1. Potential unintended areas of impact
N/A
2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A
3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
